build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-	id 'fabric-loom' version '0.4-SNAPSHOT'
+	id 'fabric-loom' version '0.5-SNAPSHOT'
```

Comment:
Is this just a general upgrade, or essential for REI support?

Reply:
Both, 0.5 for the types of dependencies when I was initially having problems and at this point no need to downgrade